Specs/DateTime/German/DateTimePeriodParser.json

```diff
@@ -48,7 +48,7 @@
       }
     ]
   },
-  {//Parser returns NULL as value. The Parser Returns Null as value. Should not be different to the one above
```

Comment:
Let's not remove these annotations unless they really no longer apply.
If they are still valid, we can move them into a json attribute called "TestComment".